bin/publish

```diff
@@ -124,6 +124,22 @@ rm -rf ui/*/.next
 pnpm install --prefer-frozen-lockfile
 git checkout pnpm-lock.yaml # the other github action will take care of this, but we need a clean working directory to publish
 
+# Check to make sure it was built correctly
+rm -rf core/dist
+if [ ! -d "core/dist" ] ||\
+   [ ! -d "cli/dist" ] ||\
+   [ ! -d "ui/ui-community/.next" ] ||\
+   [ ! -d "ui/ui-config/.next" ] ||\
+   [ ! -d "ui/ui-enterprise/.next" ] ||\
+   [ ! -d "plugins/postgres/dist" ] ||\
+   [ ! -d "plugins/mysql/dist" ] ||\
+   [ ! -d "plugins/mongo/dist" ] ||\
+   [ ! -d "plugins/bigquery/dist" ]; then
```

Comment:
Rathe then check some of these plugins, you could check them all.  There are lots of ways, but here's one suggestion:

```bash
# Count up the number of plugins with `src` folders:
PLUGINS=`find 'plugins' -name src -and -type d | wc -l`
# Count the number of dist folders in the plugins directory
BUILT_PLUGINS=`find 'plugins' -name dist -and -type d | wc -l`
# Check if they are equal
if [[ "$PLUGINS" != "$BUILT_PLUGINS" ]]; then echo 'OH NO'; fi
```